zyte_spider_templates/spiders/serp.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, Dict, Iterable, List, Optional, Union
+
+from pydantic import BaseModel, Field, field_validator
+from scrapy import Request
+from scrapy.settings import SETTINGS_PRIORITIES, BaseSettings
+from scrapy_spider_metadata import Args
+from w3lib.url import add_or_replace_parameter
+from zyte_common_items import Serp
+
+from ..params import MaxRequestsParam
+from ._google_domains import GoogleDomain
+from .base import BaseSpider
+
+
+class SearchKeywordsParam(BaseModel):
+    search_keywords: Optional[List[str]] = Field(
+        title="Search Keywords",
+        description=(
+            "Keywords to search for. Use multiple lines to trigger multiple "
+            "searches for different search keywords."
```

Comment:
It feels a bit confusing - if a single line in this item is "search keywords", then why is the whole field called "search keywords"? Would it make sense to rename the field, e.g. to "search_queries"? In this case, it's going to be a search query per line. 

"Keywords" also could be a bit imprecise. I think it implies that the order of search keywords in a query doesn't matter much, and they're not necessarily related to each other - it's just a bunch of keywords to look up in the index. I'm not sure queries are treated like this by search engines now.

Reply:
https://github.com/zytedata/zyte-spider-templates/pull/62/commits/c3a2f2369ce34aa1041fffd60a066396b77b59f9 ?